docs/integrations/microsoft-azure/web-apps.md

```diff
@@ -52,7 +62,7 @@ In this step, you configure a pipeline for shipping logs from [Azure Monitor](ht
     1. Diagnostic Settings blade will show all your existing settings if any already exist. Click **Edit Setting** if you want to change your existing settings, or click **Add diagnostic setting** to add a new one.
     1. Select the **Stream to an event hub box** checkbox.
     1. Select an Azure subscription.
-    1. **Event bub namespace.** If you have chosen Method 1 (Azure Event Hubs Source) for collecting logs, select the **EventHubNamespace** created manually, or else if you have chosen Method 2 (Collect logs from Azure monitor using Azure functions), then select `SumoAzureLogsNamespace<UniqueSuffix>` namespace created by the ARM template.
+    1. **Event hub namespace.** If you have chosen Method 1 (Azure Event Hubs Source) for collecting logs, select the **EventHubNamespace** created manually, or else if you have chosen Method 2 (Collect logs from Azure monitor using Azure functions), then select `SumoAzureLogsNamespace<UniqueSuffix>` namespace created by the ARM template.
```

Comment:
Remove this section ## Collecting logs for Azure Web Apps, we already have configure logs section later, replace it with a setup section https://help.sumologic.com/docs/integrations/microsoft-azure/azure-storage/#setup

Reply:
Updated replaced with setup instructions